Sources/Scotty/Route.swift

```diff
@@ -47,7 +47,7 @@ public struct Route<Root> {
     /// If the intended destination can be reached successfully, this function should return true. Otherwise, return false.
     ///
     /// - Parameters:
-    ///   - rootViewController: The view controller at the root of the navigation hierarchy.
+    ///   - root: The view controller at the root of the navigation hierarchy.
```

Comment:
Should this comment be changed since `root` is not necessarily a view controller anymore. Suggestion:
`///   - root: The object at the root of the navigation hierarchy.`

Reply:
Fixed